src/sentry/static/sentry/app/views/onboarding/wizardNew.jsx

```diff
@@ -0,0 +1,181 @@
+import {browserHistory} from 'react-router';
+import DocumentTitle from 'react-document-title';
+import PropTypes from 'prop-types';
+import React from 'react';
+import posed, {PoseGroup} from 'react-pose';
+import scrollToElemennt from 'scroll-to-element';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import InlineSvg from 'app/components/inlineSvg';
+import OnboardingPlatform from 'app/views/onboarding/platform';
+import OnboardingProjectSetup from 'app/views/onboarding/projectSetup';
+import OnboardingWelcome from 'app/views/onboarding/welcome';
+import PageHeading from 'app/components/pageHeading';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import withProjects from 'app/utils/withProjects';
+
+const ONBOARDING_STEPS = [
+  {
+    id: 'welcome',
+    title: t('Welcome to Sentry'),
+    component: OnboardingWelcome,
+  },
+  {
+    id: 'select-platform',
+    title: t('Platform Selection'),
+    component: OnboardingPlatform,
+  },
+  {
+    id: 'get-started',
```

Comment:
Can we make the underscore separated ids consistent with this?

Reply:
Let me actually change the other one to dashes, since these Ids here are used for url route matching.

Reply:
3e44943